### PR TITLE
Feat(T34093): add content to the accessibility page - diplanbau

### PR DESCRIPTION
**Pull request by sakutademos**
**Ticket:** https://yaits.demos-deutschland.de/T34093

**Description:** This PR adds a new translation key that is needed for the new link in the footer.

### CODE REVIEW

File: translations/messages+intl-icu.de.yml [hunk 1]
```diff
@@ -10,6 +10,7 @@ access.token: "Zugangscode"
 access.token.send: "Zugangscode versenden"
 access.token.copied: "Zugangscode wurde kopiert."
 accessibility: "Barrierefreiheit"
+accessibility.barrier.report: "Barriere melden"
```
Comment (spiess-demos) [inline translations/messages+intl-icu.de.yml:13]: According to our [conventions for composing translation keys](https://dplan-documentation.demos-europe.eu/development/guidelines-conventions/best-practices/translation-keys.html#use-dots-and-snake-case) i'd suggest to name this so:
```suggestion
accessibility.report_barrier: "Barriere melden"
```

**Review by spiess-demos — CHANGES_REQUESTED** [review]

File: translations/messages+intl-icu.de.yml (hunk 1 above)
Comment (sakutademos) [reply translations/messages+intl-icu.de.yml]: done here https://github.com/demos-europe/demosplan-core/pull/2129/commits/8b6338fa439454a2b102e419cf5cdf131aaf4d00

File: translations/messages+intl-icu.de.yml [hunk 2]
```diff
@@ -10,7 +10,7 @@ access.token: "Zugangscode"
 access.token.send: "Zugangscode versenden"
 access.token.copied: "Zugangscode wurde kopiert."
 accessibility: "Barrierefreiheit"
-accessibility.barrier.report: "Barriere melden"
+accessibility.report_barrier: "Barriere melden"
 accessibility.explanation: "Erklärung zur Barrierefreiheit"
```
Comment (spiess-demos) [inline translations/messages+intl-icu.de.yml:14]: nitpick: alphabetic order of keys

```suggestion
accessibility.explanation: "Erklärung zur Barrierefreiheit"
accessibility.report_barrier: "Barriere melden"
```

**Review by spiess-demos — APPROVED** [review]
lgtm!

**Pull request merged**